src/components/tab-bar/tab-bar.scss

```diff
@@ -4,10 +4,102 @@
 @import '@limetech/mdc-tab-indicator/mdc-tab-indicator';
 @import '@limetech/mdc-tab/mdc-tab';
 
+$tab-content-background-color: #fff;
+$tab-background-color: #f6f6f7;
+$tab-border-radius: .65rem;
+$tab-active-outer-edge-curve-size: .75rem;
+$tab-separator-width: .125rem;
+$tab-separator-background-color: #e8e8ea;
```

Comment:
Should we document these, or do we want to keep them semi-secret?

@Kiarokh @jgroth